common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -4,6 +4,10 @@
  */
 package com.wynntils.features.inventory;
 
+import static com.wynntils.models.containers.type.SearchableContainerType.BANK;
```

Comment:
Please use fully qualified names (`SearchableContainerType.BANK`) instead of static imports.